var/spack/repos/builtin/packages/ffmpeg/package.py

```diff
@@ -93,6 +100,20 @@ class Ffmpeg(AutotoolsPackage):
     depends_on("yasm@1.2.0:")
     depends_on("zlib-api")
 
+    depends_on("pkgconfig", when="+X", type="build")
+    depends_on("pkgconfig", when="+drawtext", type="build")
+    depends_on("pkgconfig", when="+libaom", type="build")
+    depends_on("pkgconfig", when="+libopenjpeg", type="build")
+    depends_on("pkgconfig", when="+libopus", type="build")
+    depends_on("pkgconfig", when="+libspeex", type="build")
+    depends_on("pkgconfig", when="+libssh", type="build")
+    depends_on("pkgconfig", when="+libvorbis", type="build")
+    depends_on("pkgconfig", when="+libwebp", type="build")
+    depends_on("pkgconfig", when="+libx264", type="build")
+    depends_on("pkgconfig", when="+libxml2", type="build")
+    depends_on("pkgconfig", when="+libzmq", type="build")
+    depends_on("pkgconfig", when="+openssl", type="build")
```

Comment:
Should we just say:
```suggestion
    depends_on("pkgconfig", type="build")
```
for brevity?

Reply:
Sure, that's fine with me. Not every build option needs it, but it's a light dependency.